mongodb-advanced/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "mongodb-advanced",
+  "version": "0.0.2",
+  "private": true,
+  "scripts": {
+    "start": "node index.js",
+    "postinstall": "node index.js compile-less"
+  },
+  "browser": "index.js",
+  "main": "index.js",
+  "dependencies": {
+    "bedrock": "^1.4.1",
+    "bedrock-angular": "^4.0.0",
+    "bedrock-angular-alert": "digitalbazaar/bedrock-angular-alert#material",
+    "bedrock-angular-material": "^1.0.0",
+    "bedrock-angular-modal": "digitalbazaar/bedrock-angular-modal#material",
+    "bedrock-express": "^2.0.6",
+    "bedrock-server": "^2.2.0",
+    "bedrock-views": "^5.0.1",
```

Comment:
Add dependency for `bedrock-webpack` and require bedrock-webpack in the index.js file.  This is so the module can be run in optimized mode.